test/bin/scenario.sh

```diff
@@ -533,13 +533,19 @@ launch_vm() {
     vm_extra_args="fips=${fips_mode}"
     vm_initrd_inject=""
 
+    declare -A os_map=(
+        ["rhel94-bootc"]="rhel9.4"
+        ["rhel-9.4"]="rhel9.4"
+        ["centos9-bootc"]="centos-stream9"
+        ["centos9"]="centos-stream9")
+
```

Comment:
Let's use switch statement to:
- Make name check more generic
- Report an error in case of no match

Reply:
done